where_to_get_help.Rmd

```diff
@@ -0,0 +1,44 @@
+# Where to get help 
+
+If you are working on R it is possible that you will come across an issue where you would need some assistance to solve it. If you require help, there are options available to seek assistance or get some feedback which are discussed in this chapter. If the question involves process or tool usage then please check the rest of this guide first as it should answer your question. Please be polite, patient, and respectful of people's time while you ask for feedback or help. 
+
+## Mailing Lists
+
+There are quite a few [mailing lists](https://www.r-project.org/mail.html) for getting help with R:
+
+  * [R-help](https://stat.ethz.ch/mailman/listinfo/r-help): 
+      * Discussions about problems and solutions using R.
+      * Announcements about the development of R.
+      * Announcements about the availability of new code.
+      * Enhancements and patches to the R source code and the R documentation.
+      * Comparison and compatibilty with S and S-plus.
+      * Posting examples and benchmarks.
+      
+  * [R-package-devel](https://stat.ethz.ch/mailman/listinfo/r-package-devel): 
+      * Getting help about package development in R.
+      * Learning about the package development process. 
+      * Discussing problems developing a package (or problem in passing the R CMD check).
+      
+  * [R-devel](https://stat.ethz.ch/mailman/listinfo/r-devel): 
+      * Questions and discussion about code development in R.
+      * Proposals of new functionality for R.
+      * Pre-testing of new versions of R.
+
+Please avoid cross-posting to both the R-package-devel and the R-devel mailing lists.
+
+## Slack

```

Comment:
ordering-wise, do we want to direct people to the R-devel slack first?

Reply:
Yes, it would be better to order that way. Thanks!

Reply:
Similarly, might better to put the R-devel mailing list next as the place for discussions requiring the attention of R Core (since more R core members on the mailing list than on Slack). Then mention R-announce for news and the other mailings list for help with using R/package development.

Reply:
Included the `R-announce` and `R-packages` mailing lists in the chapter `News and Announcements`. 
Does the order of mailing lists in this chapter look appropriate now -- `R-devel`, `R-help`, `R-package-devel`?